spawns.lua

```diff
@@ -125,18 +125,65 @@ NorthGeorgiaFARPTransportSpawns = {
     ["SE"] = {Spawner("SE FARP HELO"),nil, SEFARPLogiSpawn},
     ["MK"] = {Spawner("MK FARP HELO"), nil, MaykopLogiSpawn}
 }
+
+-- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
+
+BladderPos = {}
+BladderPos.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
+BladderPos.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+
+function BladderPos.InitOnGroup(planeGroup, vSpeed)
+	local planeGroupName	= planeGroup:getName()
+	vSpeed = vSpeed or BladderPos.vSpeed
+	
+	local clouds = env.mission.weather.clouds
+	
+	-- Calculate orbit height. Ignore for partial cloud conditions
+	local hOrbit = BladderPos.hMaxFlightAlt
+	if (((clouds.base + clouds.thickness) >= hOrbit - 305) and (clouds.density >= 7)) then
+		hOrbit = math.min(clouds.base - 305, BladderPos.hMaxFlightAlt)
```

Comment:
305 duplicated, and is a magic number. What's it represent?

Reply:
1000' (304.8m), I'll add the comments header from my original script

Reply:
Aha. I think we can then move this into its own method on `Bladderpos` (or whatever that ends up being) that takes the clouds object and spits back the appropriate height.